hub/templates/hub/area.html

```diff
@@ -39,10 +39,40 @@ <h1 class="m-0">{{ area.name }}</h1>
                 </div>
             </div>
             <div class="col-lg-9 offset-xl-1">
-
+            {% if area_type == "WMC" %}
+            <div class="card flex-grow-1 dataset-card">
```

Comment:
It’d be good to have one of these cards on the `WMC23` pages too, but going the _other_ way (ie: saying "This is a 2025 (2023?) constituency … it replaced the following 2010 constituency/ies: …"

Reply:
I'm going to fix this as part of #337